.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: python
+python:
```

Comment:
There is already a `python` in this file with version `3.6` :) . Let's update that.